chips/src/main/java/com/decathlon/vitamin/compose/chips/VitaminChipButtons.kt

```diff
@@ -0,0 +1,32 @@
+package com.decathlon.vitamin.compose.chips
+
+import androidx.compose.material.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+
+object VitaminChipButtons {
+    @Composable
+    fun Close(
+        modifier: Modifier = Modifier,
+        contentDescription: String? = null,
```

Comment:
Don't specify a default value for `contentDescription` parameters (same for Check component).

Reply:
I removed default value